pkg/plugin/basic/handler.go

```diff
@@ -25,9 +25,9 @@ func NewHandler(repo Repository) *Handler {
 // Index is the find all handler
 func (c *Handler) Index() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		span := opentracing.FromContext(r.Context(), "datastore.user.FindAll")
+		_, span := trace.StartSpan(r.Context(), "datastore.basic_auth.FindAll")
```

Comment:
the trace span should be as close as possible to the real operation name `repo.FindAll`